_toc.yml

```diff
@@ -0,0 +1,26 @@
+format: jb-book
+root: README
+parts:
+  - caption: Explore template notebooks
```

Comment:
I would put these section at the end, wdyt? also we might want to add a README about the template notebooks